### PR TITLE
Merge finished Create folder and card pages

### PR DESCRIPTION
**Pull request by ESaracay**
This PR links the card and folder creation pages with the backend.

### CODE REVIEW

File: frontend/src/main-components/CreateFolderContent.js [hunk 1]
```diff
@@ -41,7 +42,7 @@ class CreateFolderContent extends Component {
 
           <div id="formBox">
             <ul>
-              <form id="myForm" action="/folder" method="post">
+              <form id="myForm" action={`/folder?userKey=${this.userKey}`} method="post">
```
Comment (ethyus) [inline frontend/src/main-components/CreateFolderContent.js]: I think all HTTP methods are case sensitive, e.g, POST, GET, PUT, DELETE.

File: frontend/src/sub-components/folderScroll.js [hunk 2]
```diff
@@ -1,21 +1,32 @@
...
     };
     this.grabFolders();
   }
 
-  grabFolders() {
-    const folders = fetch("/folder").then((data) =>
-      useState({ isDataFetched: true, folders: data })
-    );
+  grabFolders(e) {
+    // Mount boolean ensures that grabFolders is only called once
```
Comment (tylerhou) [inline frontend/src/sub-components/folderScroll.js]: You should use componentDidMount for this.

File: frontend/src/sub-components/translate.js [hunk 3]
```diff
@@ -4,25 +4,23 @@ import privateInfo from "../../../../../credentials/token.json";
...
+      fromLang: `${fromLang}`,
+    }),
+  })
     .then((res) => res.json())
     .then((result) => {
-      return result.data.translations.translatedText;
+      return result.translation;
+    })
+    .catch((error) => {
+      return "Could not Translate";
```
Comment (tylerhou) [inline frontend/src/sub-components/translate.js]: Grammar

**Review by tylerhou — APPROVED** [review]

File: src/main/java/com/google/sps/servlets/UserCardsServlet.java [hunk 4]
```diff
@@ -78,7 +78,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         // if blobKey is null, user did not add an image
         String folderKey = request.getParameter("folderKey");
         String blobKey = getBlobKeyfromBlobstore(request, "image");
-        String labels = request.getParameter("labels");
+        //String labels = request.getParameter("labels");
```
Comment (tylerhou) [inline src/main/java/com/google/sps/servlets/UserCardsServlet.java]: Comment style (space after //)

**Pull request merged**